test/conformance/tests/sriov_operator.go

```diff
@@ -1596,3 +1564,49 @@ func waitForSRIOVStable() {
 		return isClusterReady
 	}, waitingTime, 1*time.Second).Should(BeTrue())
 }
+
+func createSriovNetworkNodePolicy(node string, sriovInfos *cluster.EnabledNodes, numVfs int, resourceName string) {
```

Comment:
createVanillaNetworkPolicy ?
Also, I would move `waitForSriovStable` outside, it goes beyond the purpose of this function.

Reply:
Done